phys/module_physics_addtendc.F

```diff
@@ -714,20 +714,6 @@ SUBROUTINE phy_bl_ten(config_flags,rk_step,n_moist,n_scalar,     &
                 ims, ime, jms, jme, kms, kme,                    &
                 its, ite, jts, jte, kts, kte                     )
 
-        if (P_QT .ge. PARAM_FIRST_SCALAR)                                         &
```

Comment:
For both P_QT and below P_QNI, since these calls are deleted, are these only ever used by MYNN

Reply:
I see for CAMUWPBLSCHEME and CAMZMSCHEME, there is also code for mixing QNI, but I don't think there needs to be if scalar_pblmix = 1.  I wonder if there is a double-counting if scalar_pblmix set to 1 or if the scalar_tendf array is just written over (I think the latter). If QNI is mixed non-locally in CAMUWPBLSCHEME and CAMZMSCHEME, perhaps the way the developers prefer it. Its probably a small impact either way.

I did a quick check for "QNI" in check_a_mundo and didn't see a check to make sure scalar_pblmix = 0 when running either of these two schemes. So, I think there is a potential to mix QNI twice, but it probably just overwrites the tendencies from the scalar_pblmix code, not resulting in a double-counting.